configure.meson

```diff
@@ -1,12 +1,11 @@
-#!/bin/sh
+#!/bin/bash
 
 # set -xv
 
-if [ -f meson.build ]; then
+if [ ! -d build ] ; then
```

Comment:
Mm, in this case I think you need
to use `$1` here. I.e. do not hard-code
the dir name.

Reply:
Or actually just create the build dir
right here?

Reply:
In fact `meson setup` will create it if necessary.

Reply:
I modded it now  to just see that the arg is not empty.